src/index.js

```diff
@@ -205,8 +205,7 @@ module.exports = (
       mainFields: ["main"],
       plugins: resolvePlugins
     },
-    // https://github.com/vercel/ncc/pull/29#pullrequestreview-177152175
-    node: false,
+    node: {},
```

Comment:
Is `{}` different than `false`?

I'm looking at the docs between v4 and v5 and the `false` value seems to be the same for both.

https://v4.webpack.js.org/configuration/node/

https://webpack.js.org/configuration/node/

Reply:
I thought it was the root cause to trigger the error but looks like it just accidently passed tests..after rerun the tests they're still failed. reverted

Reply:
wonder why CI gets passed now...seems the nodejs lib fallback wasn't the issue before? I'm still stucking at watcher test failure on my local laptop 🤔 ...